debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSchema.java

```diff
@@ -48,7 +47,7 @@
     private final TableSchemaBuilder schemaBuilder;
     private final String schemaPrefix;
     private final Tables tables;
-    private final Function<String, String> schemaNameValidator;
+    private final AvroValidator schemaNameValidator;
```

Comment:
Should not we remove the class and maybe? Validator is something that verifies that constraints are fullfilled for the value. But here we are not validating but actively converting the value to a valid one.